### PR TITLE
CLOUD-374 PMM1.0 backward compatibility test

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-749343d
client docker - perconalab/pmm-client-fb:PR-447-749343d
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-749343d.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-4e3b189
client docker - perconalab/pmm-client-fb:PR-447-4e3b189
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-4e3b189.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-53b0fc2
client docker - perconalab/pmm-client-fb:PR-447-53b0fc2
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-53b0fc2.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-1b9c593
client docker - perconalab/pmm-client-fb:PR-447-1b9c593
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-1b9c593.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-12fc23b
client docker - perconalab/pmm-client-fb:PR-447-12fc23b
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-12fc23b.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-9335a9f
client docker - perconalab/pmm-client-fb:PR-447-9335a9f
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-9335a9f.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-ff0d5f4
client docker - perconalab/pmm-client-fb:PR-447-ff0d5f4
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-ff0d5f4.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-38052a4
client docker - perconalab/pmm-client-fb:PR-447-38052a4
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-38052a4.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-4335ef3
client docker - perconalab/pmm-client-fb:PR-447-4335ef3
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-4335ef3.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-5b3c26d
client docker - perconalab/pmm-client-fb:PR-447-5b3c26d
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-5b3c26d.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-a40b889
client docker - perconalab/pmm-client-fb:PR-447-a40b889
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-a40b889.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-3b143e9
client docker - perconalab/pmm-client-fb:PR-447-3b143e9
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-3b143e9.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-c494ec3
client docker - perconalab/pmm-client-fb:PR-447-c494ec3
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-c494ec3.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-552a0fc
client docker - perconalab/pmm-client-fb:PR-447-552a0fc
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-552a0fc.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-552a0fc
client docker - perconalab/pmm-client-fb:PR-447-552a0fc
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-552a0fc.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-552a0fc
client docker - perconalab/pmm-client-fb:PR-447-552a0fc
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-552a0fc.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-5d07797
client docker - perconalab/pmm-client-fb:PR-447-5d07797
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-5d07797.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-641807c
client docker - perconalab/pmm-client-fb:PR-447-641807c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-641807c.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-d2e10a9
client docker - perconalab/pmm-client-fb:PR-447-d2e10a9
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-d2e10a9.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-22916b7
client docker - perconalab/pmm-client-fb:PR-447-22916b7
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-22916b7.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-6bec144
client docker - perconalab/pmm-client-fb:PR-447-6bec144
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-6bec144.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-657c44f
client docker - perconalab/pmm-client-fb:PR-447-657c44f
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-657c44f.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-256274d
client docker - perconalab/pmm-client-fb:PR-447-256274d
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-256274d.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-53fb750
client docker - perconalab/pmm-client-fb:PR-447-53fb750
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-53fb750.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-41a995e
client docker - perconalab/pmm-client-fb:PR-447-41a995e
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-41a995e.tar.gz

File: build/pmm-client-docker/entrypoint.sh [hunk 1]
```diff
@@ -35,54 +35,143 @@ wait_for_port() {
...
+
+    PMM_SERVER_IP=$(ping -c 1 "${PMM_SERVER/:*/}" | grep PING | sed -e 's/).*//; s/.*(//')
+    SRC_ADDR=$(ip route get "${PMM_SERVER_IP}" | grep 'src ' | sed -e 's/.* src //; s/ .*//')
+    CLIENT_NAME="${HOSTNAME}"
+
+    SERVER_RESPONSE_CODE=$(curl -k -s -o /dev/null -w "%{http_code}" "https://${PMM_USER}:${PMM_PASSWORD}@${PMM_SERVER}/v1/readyz")
+
+    if [[ "${SERVER_RESPONSE_CODE}" == '200' ]]; then
+        export PATH="/usr/local/percona/pmm2/bin/:${PATH}"
+        pmm2_start "$@"
```
Comment (delgod) [inline build/pmm-client-docker/entrypoint.sh]:
```suggestion
        pmm2_start
```

File: build/pmm-client-docker/entrypoint.sh [hunk 2]
```diff
@@ -35,54 +35,143 @@ wait_for_port() {
...
+                pmm-admin add "${DB_TYPE}" \
+                            --skip-connection-check \
+                            --username "${MONGODB_USER}" \
+                            --password "${MONGODB_PASSWORD}" \
+                            --replication-set "${DB_REPLSET}"\
+                            --server-url="https://${PMM_USER}:${PMM_PASSWORD}@${PMM_SERVER}/" \
+                            --server-insecure-tls \
+                            --cluster "${DB_CLUSTER}" \
+                            --use-profiler \
+                            "${CLIENT_NAME}":27017 \
```
Comment (delgod) [inline build/pmm-client-docker/entrypoint.sh]:
```suggestion
                            ${DB_HOST}:${DB_PORT}
```

File: build/pmm-client-docker/entrypoint.sh [hunk 3]
```diff
@@ -35,54 +35,143 @@ wait_for_port() {
...
+
+    cat /usr/local/percona/pmm2/pmm-agent-tmp.log
+
+    if [ -n "${DB_TYPE}" ]; then
+        case "${DB_TYPE}" in
+            mongodb )
+                pmm-admin add "${DB_TYPE}" \
+                            --skip-connection-check \
+                            --username "${MONGODB_USER}" \
+                            --password "${MONGODB_PASSWORD}" \
```
Comment (delgod) [inline build/pmm-client-docker/entrypoint.sh]: username/password passed via DB_ARGS
```suggestion
                            ${DB_ARGS} \
```

File: build/pmm-client-docker/entrypoint.sh [hunk 4]
```diff
@@ -35,54 +35,143 @@ wait_for_port() {
...
+                            --server-url="https://${PMM_USER}:${PMM_PASSWORD}@${PMM_SERVER}/" \
+                            --server-insecure-tls \
+                            ${DB_ARGS} \
+                            ${DB_HOST}:${DB_PORT}
+                ;;
+        esac
+    fi
+
+    kill %1
+    exec pmm-agent --config-file=/usr/local/percona/pmm2/config/pmm-agent.yaml --ports-min=${CLIENT_PORT_MIN:-30100} --ports-max=${CLIENT_PORT_MAX:-30200}
```
Comment (delgod) [inline build/pmm-client-docker/entrypoint.sh]: minor style improvement
```suggestion
    exec pmm-agent \
        --config-file=/usr/local/percona/pmm2/config/pmm-agent.yaml \
        --ports-min=${CLIENT_PORT_MIN:-30100} \
        --ports-max=${CLIENT_PORT_MAX:-30200}
```

File: build/pmm-client-docker/entrypoint.sh [hunk 5]
```diff
@@ -35,54 +35,143 @@ wait_for_port() {
...
-    --skip-root \
+pmm2_start() {
+    AGENT_CONFIG_FILE=/usr/local/percona/pmm2/config/pmm-agent.yaml
+    rm -f ${AGENT_CONFIG_FILE}
+    if [ -n "${PMM_USER}" ]; then
+        ARGS+=" --server-username=${PMM_USER}"
+    fi
+
+    pmm-agent setup \
     --force \
```
Comment (delgod) [inline build/pmm-client-docker/entrypoint.sh]: style fix - tabulation (4 spaces added)
```suggestion
        --force \
```

File: build/pmm-client-docker/entrypoint.sh [hunk 6]
```diff
@@ -35,54 +35,143 @@ wait_for_port() {
...
+        echo PMM_SERVER is not specified. exiting
+        exit 1
+    fi
+    if [ -n "${PMM_PASSWORD}" ]; then
+        ARGS+=" --server-password=${PMM_PASSWORD}"
+    fi
+
+    PMM_SERVER_IP=$(ping -c 1 "${PMM_SERVER/:*/}" | grep PING | sed -e 's/).*//; s/.*(//')
+    SRC_ADDR=$(ip route get "${PMM_SERVER_IP}" | grep 'src ' | sed -e 's/.* src //; s/ .*//')
+    CLIENT_NAME="${HOSTNAME}"
```
Comment (delgod) [inline build/pmm-client-docker/entrypoint.sh]:
```suggestion
    CLIENT_NAME="${CLIENT_NAME:-$HOSTNAME}"
```

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-ed8914a
client docker - perconalab/pmm-client-fb:PR-447-ed8914a
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-ed8914a.tar.gz

File: build/pmm-client-docker/entrypoint.sh [hunk 7]
```diff
@@ -35,54 +35,143 @@ wait_for_port() {
...
+        case "${DB_TYPE}" in
+            mongodb )
+                pmm-admin add "${DB_TYPE}" \
+                            --skip-connection-check \
+                            --username "${MONGODB_USER}" \
+                            --password "${MONGODB_PASSWORD}" \
+                            --replication-set "${DB_REPLSET}"\
+                            --server-url="https://${PMM_USER}:${PMM_PASSWORD}@${PMM_SERVER}/" \
+                            --server-insecure-tls \
+                            --cluster "${DB_CLUSTER}" \
```
Comment (delgod) [inline build/pmm-client-docker/entrypoint.sh]: cluster passed via DB_ARGS

File: build/pmm-client-docker/entrypoint.sh [hunk 8]
```diff
@@ -35,54 +35,143 @@ wait_for_port() {
...
+    cat /usr/local/percona/pmm2/pmm-agent-tmp.log
+
+    if [ -n "${DB_TYPE}" ]; then
+        case "${DB_TYPE}" in
+            mongodb )
+                pmm-admin add "${DB_TYPE}" \
+                            --skip-connection-check \
+                            --username "${MONGODB_USER}" \
+                            --password "${MONGODB_PASSWORD}" \
+                            --replication-set "${DB_REPLSET}"\
```
Comment (delgod) [inline build/pmm-client-docker/entrypoint.sh]: it is better to pass `replication-set` via  passed via DB_ARGS

File: build/pmm-client-docker/entrypoint.sh [hunk 9]
```diff
@@ -35,54 +35,143 @@ wait_for_port() {
...
+                            --server-url="https://${PMM_USER}:${PMM_PASSWORD}@${PMM_SERVER}/" \
+                            --server-insecure-tls \
+                            --cluster "${DB_CLUSTER}" \
+                            --use-profiler \
```
Comment (delgod) [inline build/pmm-client-docker/entrypoint.sh]: can be passed via DB_ARGS in PSMDB Operator

File: build/pmm-client-docker/entrypoint.sh (hunk 6 above)
Comment (cap1984) [reply build/pmm-client-docker/entrypoint.sh]: done

File: build/pmm-client-docker/entrypoint.sh (hunk 1 above)
Comment (cap1984) [reply build/pmm-client-docker/entrypoint.sh]: done

File: build/pmm-client-docker/entrypoint.sh (hunk 4 above)
Comment (cap1984) [reply build/pmm-client-docker/entrypoint.sh]: done

File: build/pmm-client-docker/entrypoint.sh (hunk 2 above)
Comment (cap1984) [reply build/pmm-client-docker/entrypoint.sh]: done

File: build/pmm-client-docker/entrypoint.sh (hunk 9 above)
Comment (cap1984) [reply build/pmm-client-docker/entrypoint.sh]: done

File: build/pmm-client-docker/entrypoint.sh (hunk 7 above)
Comment (cap1984) [reply build/pmm-client-docker/entrypoint.sh]: done

File: build/pmm-client-docker/entrypoint.sh (hunk 8 above)
Comment (cap1984) [reply build/pmm-client-docker/entrypoint.sh]: done

File: build/pmm-client-docker/entrypoint.sh (hunk 3 above)
Comment (cap1984) [reply build/pmm-client-docker/entrypoint.sh]: done

File: build/pmm-client-docker/entrypoint.sh (hunk 5 above)
Comment (cap1984) [reply build/pmm-client-docker/entrypoint.sh]: done

File: build/pmm-client-docker/Dockerfile [hunk 10]
```diff
@@ -2,19 +2,35 @@ FROM centos:7
...
+ENV PMM1_DIR="/usr/local/percona/pmm-client"
+ENV QAN1_DIR="/usr/local/percona/qan-agent"
+
+RUN curl -o /tmp/pmm-client.tar.gz https://www.percona.com/downloads/pmm/1.17.1/binary/tarball/pmm-client-1.17.1.tar.gz
+RUN tar -zxpf /tmp/pmm-client.tar.gz -C /tmp \
+    && mkdir -p $PMM1_DIR/textfile_collector \
+    && mkdir -p $QAN1_DIR/bin \
+    && cd /tmp/pmm-client-* \
+    && bash ./install \
+    && ln -s /usr/bin/pid-watchdog /usr/bin/service \
```
[the remark this answers is not in the document]
Comment (cap1984) [reply build/pmm-client-docker/Dockerfile]: done

File: build/pmm-client-docker/Dockerfile [hunk 11]
```diff
@@ -2,19 +2,35 @@ FROM centos:7
 RUN yum -y install iproute nc \
     && rm -rf /var/cache/yum
 
+ENV PMM1_DIR="/usr/local/percona/pmm-client"
+ENV QAN1_DIR="/usr/local/percona/qan-agent"
+
+RUN curl -o /tmp/pmm-client.tar.gz https://www.percona.com/downloads/pmm/1.17.1/binary/tarball/pmm-client-1.17.1.tar.gz
+RUN tar -zxpf /tmp/pmm-client.tar.gz -C /tmp \
+    && mkdir -p $PMM1_DIR/textfile_collector \
+    && mkdir -p $QAN1_DIR/bin \
```
[the remark this answers is not in the document]
Comment (cap1984) [reply build/pmm-client-docker/Dockerfile]: done

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-e56c615
client docker - perconalab/pmm-client-fb:PR-447-e56c615
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-e56c615.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-8902e06
client docker - perconalab/pmm-client-fb:PR-447-8902e06
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-8902e06.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-d31579c
client docker - perconalab/pmm-client-fb:PR-447-d31579c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-d31579c.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-f9fc551
client docker - perconalab/pmm-client-fb:PR-447-f9fc551
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-f9fc551.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-86f8839
client docker - perconalab/pmm-client-fb:PR-447-86f8839
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-86f8839.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-46d2698
client docker - perconalab/pmm-client-fb:PR-447-46d2698
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-46d2698.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-62c48c4
client docker - perconalab/pmm-client-fb:PR-447-62c48c4
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-62c48c4.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-515a392
client docker - perconalab/pmm-client-fb:PR-447-515a392
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-515a392.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-5caeb6d
client docker - perconalab/pmm-client-fb:PR-447-5caeb6d
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-5caeb6d.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-5a2e645
client docker - perconalab/pmm-client-fb:PR-447-5a2e645
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-5a2e645.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-6263ed4
client docker - perconalab/pmm-client-fb:PR-447-6263ed4
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-6263ed4.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-58d5c51
client docker - perconalab/pmm-client-fb:PR-447-58d5c51
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-58d5c51.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-470c8e7
client docker - perconalab/pmm-client-fb:PR-447-470c8e7
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-470c8e7.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-8136f0a
client docker - perconalab/pmm-client-fb:PR-447-8136f0a
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-8136f0a.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-3e77c9a
client docker - perconalab/pmm-client-fb:PR-447-3e77c9a
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-3e77c9a.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-b241307
client docker - perconalab/pmm-client-fb:PR-447-b241307
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-b241307.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-6e290e4
client docker - perconalab/pmm-client-fb:PR-447-6e290e4
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-6e290e4.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-4782b51
client docker - perconalab/pmm-client-fb:PR-447-4782b51
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-4782b51.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-81530f0
client docker - perconalab/pmm-client-fb:PR-447-81530f0
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-81530f0.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-442129b
client docker - perconalab/pmm-client-fb:PR-447-442129b
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-442129b.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-9aa5e74
client docker - perconalab/pmm-client-fb:PR-447-9aa5e74
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-9aa5e74.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-94d6959
client docker - perconalab/pmm-client-fb:PR-447-94d6959
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-94d6959.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-2150807
client docker - perconalab/pmm-client-fb:PR-447-2150807
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-2150807.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-912e666
client docker - perconalab/pmm-client-fb:PR-447-912e666
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-912e666.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-d1abdaa
client docker - perconalab/pmm-client-fb:PR-447-d1abdaa
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-d1abdaa.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-b9674ae
client docker - perconalab/pmm-client-fb:PR-447-b9674ae
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-b9674ae.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-7fd5cba
client docker - perconalab/pmm-client-fb:PR-447-7fd5cba
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-7fd5cba.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-447-022e96c
client docker - perconalab/pmm-client-fb:PR-447-022e96c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-447-022e96c.tar.gz

**Pull request closed without merging**